sql.txt

```diff
@@ -1,39 +1,30 @@
 (1) Using the Customer Table, select the CustomerName, ContactName, and Country.  Use ORDER BY to order by Country.  
 Use LIMIT and OFFSET to get entries 11 through 20. Paste your SQL statement below. 
 
-SELECT CustomerName, ContactName, Country FROM Customers ORDER BY COUNTRY LIMIT 10 OFFSET 10;
-
-
+SELECT CustomerName, ContactName, Country FROM Customers ORDER BY Country LIMIT 10 OFFSET 10;
 
 (2) Select all columns from the Customer table where the ContactName starts with A.  Paste your SQL statement below.
 
 SELECT * FROM Customers WHERE ContactName LIKE "A%";
 
-
-
 (3) Select all columns from the OrderDetails table where the ProductID is 51 and the quantity is greater than 10.
 Paste your SQL statement below.
 
 SELECT * FROM OrderDetails WHERE ProductID = 51 AND Quantity > 10;
 
-
-
 (4) Insert 3 rows into the Products table.  Note that you will have to specify a valid SupplierID and CategoryID, 
 corresponding to rows from the Supplier and Category tables.  Paste your three SQL statements below.
 
-INSERT INTO Products (SupplierID, CategoryID, ProductName, Price, Unit) 
-	VALUES (5, 7, 'Red Mill Instant Oats', 7.99, '1 - 1 lb bag'),
-    		(3, 7, 'Sperry Springs Steel Cut Oats', 8.50, '1 - 1 lb can'),
-            (1, 2, 'Bear Creek Lentils', 4.99, '4 - 16 oz cans')
-
-
+INSERT INTO Products (ProductName, SupplierID, CategoryID, Unit, Price)
```

Comment:
Nice getting it into a single statement! Just noting though that this might not always be the best strategy for later debugging.